setup.sh

```diff
@@ -1,18 +1,6 @@
 #!/bin/bash
-bazel build //src:train_classifier_g
-
-# check if spacy en is linked
-SPACY_PATH=bazel-bin/src/train_classifier_g.runfiles/deps_pypi__spacy_2_2_4/spacy/data/
```

Comment:
ooh, is this not necessary any more? nice! if that's the case, I would say you can drop setup.sh -- it's clear enough how to build the project

Reply:
deleted.